functions/src/CVE/index.ts

```diff
@@ -21,8 +22,13 @@ app.get('/cves', (request, response) => {
     const spl1 = request.query.spl1;
     const spl2 = request.query.spl2;  
     const androidVersion = request.query.androidVersion;
+    const v1 = request.query.v1; 
```

Comment:
In the catch() statement we should return error 500. Here we should do data validation (CVE format), and return error 400. As a bonus, we could return error 404 if CVE requested is not present.